core/src/test/java/io/questdb/test/jit/CompiledFilterIRSerializerTest.java

```diff
@@ -847,9 +847,16 @@ public void testInSubSelect() throws Exception {
         serialize("asymbol in (select asymbol from tab limit 1)");
     }
 
-    @Test(expected = SqlException.class)
+    @Test
     public void testTimestampInLiteral() throws Exception {
         serialize("atimestamp in '2020-01-01'");
+        assertIR("(i64 1577836800000000L)(i64 atimestamp)(>=)(i64 1577923199999999L)(i64 atimestamp)(<=)(&&)(ret)");
+        serialize("atimestamp in '2020-01-01;15s'");
+        assertIR("(i64 1577836800000000L)(i64 atimestamp)(>=)(i64 1577923214999999L)(i64 atimestamp)(<=)(&&)(ret)");
+        serialize("atimestamp in '2020-01-01T23:59:58;4s;-1d;3'");
```

Comment:
Would be great to add a test with a more complex filter involving other predicates, e.g. `along = 42 or atimestamp in '2020-01-01T23:59:58;4s;-1d;3'`

Reply:
Added in last commit